pylint_django/tests/input/migrations/0004_with_backwards.py

```diff
@@ -0,0 +1,19 @@
+# pylint: disable=missing-docstring, invalid-name
+from django.db import migrations
+
+
+# pylint: disable=unused-argument
+def forwards_test(apps, schema_editor):
+    pass
+
+
+# pylint: disable=unused-argument
+def backwards_test(apps, schema_editor):
+    pass
+
+
+class Migration(migrations.Migration):
+
+    operations = [
+        migrations.RunPython(forwards_test, backwards_test)
+    ]
```

Comment:
@brymut you also need to provide  all of the calling variantions, incl. only as `RunPython()` to make sure all of the conditionals in the checker work as expected.